### PR TITLE
Build admin plugin in CI

### LINKED ISSUE #1707: Run `npm install` in TravisCI to test kinto/plugins/admin/package.json
Currently we don't test `npm install` of `kinto/plugins/admin/package.json` in CI.

Appart from the release step (see `kinto/plugins/admin/release_hook.py`), we never run it.

I believe we could add a TravisCI task that runs `make build-kinto-admin`

https://github.com/Kinto/kinto/blob/be8c22bf15189f5afa15592edc53535454093c61/Makefile#L64-L65

### PR DESCRIPTION
**Pull request by itaisteinherz**
I've closed #1855 as it got messy for some reason, and opened this PR instead. This fixes #1707.

- [x] Add tests.
- [x] Add a changelog entry.
- [x] Add your name in the contributors file.

### CODE REVIEW

**Comment by itaisteinherz** [comment]
@leplatrem Continuing the discussion from https://github.com/Kinto/kinto/pull/1855#issuecomment-434628803:
> I think the admin build should have a dedicated entry in the matrix instead.

I'll add a dedicated entry to the matrix.

> Is that what you say that fails?

No, what I meant is that running (from the project root):
```bash
cd kinto/plugins/admin/; npm run test
```
fails.

**Comment by itaisteinherz** [comment]
@leplatrem I ended up using a simple bash check to build the plugin only on the main matrix entry (Python 3.7). 47c6ef03317277aa1cf3f5fc9ab9625c281e012a

**Comment by Natim** [comment]
@itaisteinherz why did you pick this option rather than adding something to the matrix?

**Comment by Natim** [comment]
Thank you for your work on this @itaisteinherz

**Comment by itaisteinherz** [comment]
> @itaisteinherz why did you pick this option rather than adding something to the matrix?

As I wrote above:

> This keeps the CI build as simple as possible and reduces unnecessary complications.

For every entry in the matrix, Travis has to start a container, install all of the necessary packages and frameworks on it, and finally start Kinto, and only then can it start the tests. Avoiding creating additional matrix entries makes the CI faster.

File: .travis.yml [hunk 1]
```diff
@@ -45,3 +45,9 @@ matrix:
       script:
         - make runkinto & sleep 5
         - make functional
+    - python: 3.7
```
Comment (itaisteinherz) [inline .travis.yml]: Why was this added? @Natim

**Comment by Natim** [comment]
> Avoiding creating additional matrix entries makes the CI faster.

Maybe but it also makes it more harder to understand what failed.
